### PR TITLE
Breadth First Search in Python

### PR DESCRIPTION
**Pull request by Pranav016**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added Graph traversal algorithm of Breadth First Search in Python.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1528

### CODE REVIEW

**Comment by shraddhavp** [comment]
Please format it using http://pep8online.com/

File: Python/graphs/BFS.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# 0 1
+# 0 3
+# 1 2
+# 2 3
+# 1 3
+
+# Sample output-
+# 3 0 1
+
+# Time Complexity: O(V+E) where V is number of vertices in the graph and E is number of edges in the graph.
```
Comment (charlie219) [inline Python/graphs/BFS.py]: Use multi-line Comment instead of single-lined 👍

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Pranav016 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Review by charlie219 — CHANGES_REQUESTED** [review]

File: Python/graphs/BFS.py (hunk 1 above)
Comment (Pranav016) [reply Python/graphs/BFS.py]: Done 👍

**Comment by Pranav016** [comment]
Can you please review this? @iamrajiv @atarax665 @shraddhavp
Thanks

**Comment by shraddhavp** [comment]
👍

**Review by charlie219 — APPROVED** [review]
LGTM 👍

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Pranav016 Your code is not according to PEP 8 rules.

**Comment by Pranav016** [comment]
> @Pranav016 Your code is not according to PEP 8 rules.

Done 😄

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**